### DISCUSSION TITLE
[功能请求] 轨迹箭头不与鼠标指针重合

### DISCUSSION THREAD

**Discussion by xmbai9507** [shuax/MouseInc.Settings#288] [Ideas]
如题, 目前二者是重合的, 箭头比较不显眼. 
![image](https://user-images.githubusercontent.com/65625632/173174972-c9fd0b66-9aa8-4e5f-b4d1-94001880f302.png)

如果箭头在位置上能落后指针一点, 显示效果会更好. 或者把箭头部分加长.
![image](https://user-images.githubusercontent.com/65625632/173175013-99425204-62b7-4cf8-8de9-a48316778177.png)

**Comment by shuax** [comment]
不重合有些人会觉得有延迟，这个就不改了。
